subsys/bluetooth/controller/hci_internal.c

```diff
@@ -310,6 +310,77 @@ static void supported_commands(sdc_hci_ip_supported_commands_t *cmds)
 #endif
 }
 
+#if defined(CONFIG_BT_HCI_VS)
+static void vs_supported_commands(sdc_hci_vs_zephyr_supported_commands_t *cmds)
+{
+	memset(cmds, 0, sizeof(*cmds));
+
+	cmds->read_version_info = 1;
+	cmds->read_supported_commands = 1;
+
+#if defined(CONFIG_BT_HCI_VS_EXT)
+	cmds->write_bd_addr = 1;
+	cmds->read_static_addresses = 1;
+	cmds->read_chip_temperature = 1;
+
+#if defined(CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL)
+	cmds->write_tx_power_level = 1;
+#endif /* CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL */
+#endif /* CONFIG_BT_HCI_VS_EXT */
+}
+#endif	/* CONFIG_BT_HCI_VS */
+
+static void supported_features(sdc_hci_ip_lmp_features_t *features)
+{
+	memset(features, 0, sizeof(*features));
+
+	features->bdedr_not_supported = 1;
+	features->le_supported = 1;
+}
+
+static void le_supported_features(sdc_hci_le_le_features_t *features)
+{
+	memset(features, 0, sizeof(*features));
+
+#ifdef CONFIG_BT_CTLR_LE_ENC
+	features->le_encryption = 1;
+#endif
+
+#ifdef CONFIG_BT_CTLR_EXT_REJ_IND
+	features->extended_reject_indication = 1;
+#endif
+
```

Comment:
Looks like slave feature request should be supported, it is enabled by BT_PERIPHERAL from what I can tell.

Reply:
Slave initiated feature request is not supported by the SoftDevice Controller

Reply:
Ok, my mistake.